ykpack/src/types.rs

```diff
@@ -375,23 +350,19 @@ impl Display for SignedInt {
 /// A call target.
 #[derive(Serialize, Deserialize, PartialEq, Eq, Debug, Clone)]
 pub enum CallOperand {
-    /// A statically known function identified by its DefId.
-    /// A pair: the definition ID and the binary symbol name, if known. If the callee doeesn't have
-    /// all of its type parameters instantiated, then there will be no symbol.
-    Fn(DefId, Option<String>),
+    /// A call to a binary symbol.
+    Fn(Option<String>), // FIXME get rid of option.
```

Comment:
I think we can fix this in this PR.

Reply:
Do you want to do that and then I can review things together?

Reply:
49ea6a7 is all that is required. There's no compiler-side change, as we don't yet instantiate any calls in SIR.

Reply:
OK.